pkg/testutils/modulecr.go

```diff
@@ -126,3 +128,45 @@ func ModuleCRIsInExpectedState(ctx context.Context,
 	}
 	return state == string(expectedState)
 }
+
+func ModuleDeploymentExists(ctx context.Context,
+	clnt client.Client,
+	namespace string,
+	deploymentName string,
+) bool {
+	var deployment appsv1.Deployment
+	err := clnt.Get(ctx, client.ObjectKey{
+		Namespace: namespace,
+		Name:      deploymentName,
+	}, &deployment)
+
+	return err == nil && deployment.Status.AvailableReplicas != 0
+}
+
+func ChangeKymaModuleChannel(ctx context.Context,
```

Comment:
This function better put it in /testutils/kyma.go, and if you put there, you can find there is already similar one exists, `UpdateKymaModuleChannel`